docs/view-default.md

```diff
@@ -321,15 +321,17 @@ orb.events.on(OrbEventType.MOUSE_CLICK, (event) => {
 
 ### Property `interaction`
 
-The optional property `interaction` has one property that you can enable/disable:
+The optional property `interaction` has two properties that you can enable/disable:
```

Comment:
In this file, lines `:157` and `:85` are missing the mention of the `isZoomEnabled`.

Reply:
Added `isZoomEnabled` pls check